rfcs/003-randomness.md

```diff
@@ -0,0 +1,151 @@
+# RFC: Randomness
+
+## Abstract
+
+This document covers existing randomness methods in Polkadot and discusses which
+would be appropriate to the Polka Storage project.
+
+## Introduction
+
+The Polka Storage project requires randomness as a form of creating challenges
+for Storage Providers, with the aim of stopping them from discarding the files
+they are supposed to store, by pre-computing proofs for upcoming challenges.
+Achieving this randomness is not trivial — for example, we cannot allow the
+Storage Provider to generate the randomness alone since they may be dishonest
+and generate values that allow them to fake the storage proofs; furthermore by
+their very nature, blockchains must be deterministic and as such, randomness
+methods achieved on-chain are not suitable for all use cases.
```

Comment:
` randomness
methods achieved on-chain are not suitable for all use cases.` 
What do you mean by that?